### DISCUSSION TITLE
Track system call duration

### DISCUSSION THREAD

**Discussion by skandbug** [aquasecurity/tracee#3296] [Questions and Help]
Tracee is a great tool.
But what method (command) do I use to track the working time of a certain system call and the length of memory read in a single process?

Environment:
Use Tracee to track a certain PID and get a series of sequential systems. At present, you can see timestamp (nanoseconds), system call parameters, etc.
```shell
~/tracee -f pid=9515 -f set=syscalls -f follow --output option:parse-arguments --output json:/tmp/tracee/tracee_test.json
```

Questions are as follows:
1. The time consumed by the work of each system call cannot be obtained (it is not the subtraction of the timestamps of two adjacent system calls), what command can I use to obtain it?
2. If the return value of the second parameter of read is "0xffffafd73dc9", how to know the length.

Thanks in advance for your answer.

**Comment by yanivagman** [comment]
Tracee is a security focused tool and is not intended to be used as a performance tool, thus information such as system call duration is not provided by Tracee
